additional_tasks/task_11.py

```diff
@@ -0,0 +1,13 @@
+
+def show_max_num(*args: int):
+    return max(args)
+
+
+def main():
+    user_first_num = int(input("Enter first num: "))
+    user_second_num = int(input("Enter second num: "))
+    biggest_value = show_max_num(user_first_num, user_second_num)
+    print(f'{biggest_value} is bigger')
+
+
+main()
```

Comment:
Якщо вже робити точку входу до скрипта, то не слід забувати про перевірку: виконується поточний скрипт, чи просто відбувся імпорт з нього деінде.

```suggestion
if __name__ == "__main__":
    main()
```